packages/utils/src/types.ts

```diff
@@ -541,7 +541,7 @@ export type DescriptionFieldProps<T = any, S extends StrictRJSFSchema = RJSFSche
   /** The uiSchema object for this description field */
   uiSchema?: UiSchema<T, S, F>;
   /** The description of the field being rendered */
-  description: string | ReactElement;
+  description: string;
```

Comment:
I believe this wants to be `ReactNode` as well. Which is what is causing the `@rjsf/core` tests to fail